### PR TITLE
EZ Alt Title: Investigation Agent

### PR DESCRIPTION
**Pull request by projectkepler-ru**
## About the Pull Request

Alt Title for EZ agents this also give them a new 9mm revolver, the Rhino. alongside the detective outfit to go with
## Why It's Good For The Game

SD shouldn't be half arsed to go everywhere himself when he has his trust-worthy officer to investigate into all the issues
## Changelog

:cl:
add: Added new title + gun + outfit, EZ investigators
imageadd: added some icons and images
/:cl:

### CODE REVIEW

File: maps/site53/job/outfits.dm [hunk 1]
```diff
@@ -27,6 +27,20 @@
...
+	shoes = /obj/item/clothing/shoes/dress
+	suit = /obj/item/clothing/suit/storage/det_trench
+	id_types = list(/obj/item/card/id/adminlvl4)
+	l_ear = /obj/item/device/radio/headset/heads/hop
+	back = /obj/item/storage/backpack/satchel/pocketbook
+	belt = /obj/item/storage/belt/holster/security/fullrhino
+	l_hand = /obj/item/storage/briefcase/crimekit
+	glasses = /obj/item/clothing/glasses/sunglasses/sechud/goggles
+	head = /obj/item/clothing/head/det
+	gloves = /obj/item/clothing/gloves/forenic
```
Comment (hry-gh) [inline maps/site53/job/outfits.dm]:
```suggestion
	gloves = /obj/item/clothing/gloves/forensic
```

**Comment by checkraisefold** [comment]
having the HRO be the detective is kind of dumb, someone please just add a detective role

File: code/game/objects/items/weapons/storage/belt.dm [hunk 2]
```diff
@@ -319,6 +319,19 @@
 	new /obj/item/handcuffs(src)
 	update_icon()
 
+/obj/item/storage/belt/holster/security/fullrhino/New()
```
Comment (ivanmixo) [inline code/game/objects/items/weapons/storage/belt.dm]: Could you do it in `Initialize()` instead? Be sure to do `. = ..()` though

**Review by ivanmixo — CHANGES_REQUESTED** [review]

**Comment by projectkepler-ru** [comment]
> having the HRO be the detective is kind of dumb, someone please just add a detective role

invalid opinion but i changed anywy

**Comment by checkraisefold** [comment]
image
image
fix this

**Comment by hry-gh** [comment]
as above + conflicts

**Comment by hry-gh** [comment]
messy pr, please re-open if still interested

**Pull request closed without merging**